scenario/tasks.py

```diff
@@ -1,8 +1,17 @@
 from mhq_web.celery import app
-from scenario.models import Scenario
+from .models import Scenario
+from django.shortcuts import get_object_or_404
+from .execute import Execution
+from request.managers import RequestExecution
+import requests
 
 
 @app.task
-def execute(scenario_id):
-    scenario = Scenario.objects.get(id=scenario_id)
-    print('Hi')
+def execute_scenario():
+    res = requests.post('https://60c1f3514f7e880017dc0e65.mockapi.io/scenario')
+    print(res)
```

Comment:
ow in asan be scenario execution vasl nashode

Reply:
re dg fln be celery vasl nashode fuck dockerrrr